core/embed/bootloader/version_check.h

```diff
@@ -23,15 +23,18 @@
 #include "secbool.h"
 
 // Protection against bootloader downgrade
-// This functions checks if the bootloader version is at least the minimum
-// required version The function also updates stored bootloader version
+
+// Ensures bootloader version is stored in monotonic counter
+// If the version cannot be written, the function will shutdown the device
 void ensure_bootloader_min_version(void);
 
 // Protection against firmware downgrade
+
 // This functions checks if the firmware version is at least the minimum
 // required version, returns sectrue if check_version is higher or equal to the
 // stored version
 secbool check_firmware_min_version(uint8_t check_version);
 
-// Update the minimum firmware version stored in monotonic counter
-void update_firmware_min_version(uint8_t version);
\ No newline at end of file
+// Ensures firmware version is stored in monotonic counter
+// If the version cannot be written, the function will shutdown the device
+void ensure_firmware_min_version(uint8_t version);
```

Comment:
no newline at end of file -- surprised that this passes editor check...?

Reply:
it didn't https://github.com/trezor/trezor-firmware/pull/4133/commits/7583628226dbd6ccb186890124b400cc1ca28ed1